examples/gallery/histograms/scatter_and_histograms.py

```diff
@@ -13,15 +13,16 @@
 import numpy as np
 import pygmt
 
-np.random.seed(19680801)
-
 # Generate random data from a standard normal distribution centered on 0
-x = np.random.randn(1000)
-y = np.random.randn(1000)
+# with a standard deviation of 1
+rng = np.random.default_rng(seed=19680801)
+x = rng.normal(0, 1, 1000)
+y = rng.normal(0, 1, 1000)
```

Comment:
Use keyword arguments to be clearer what the arguments mean?
```suggestion
y = rng.normal(loc=0, scale=1, size=1000)
x = rng.normal(loc=0, scale=1, size=1000)
```

Reply:
I updated all examples to use the keyword arguments to have this consistent.

Reply:
Thanks, no worries.